src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/impl/OrganisationIdentifierValidatorImpl.java

```diff
@@ -119,4 +126,33 @@ public void validateOrganisationExistsAndActive(String orgId) {
         }
         validateOrganisationIsActive(org.get(), BAD_REQUEST);
     }
+
+    public void validateGetRefreshUsersParams(String since, String userId, Integer size) {
+        if ((since == null && userId == null) || (since != null && userId != null)) {
```

Comment:
updated variable name to `pageSize` in commit [8d31f37](https://github.com/hmcts/rd-professional-api/pull/1529/commits/8d31f37d573af219fe45f3c2997629cf574399f9)

re `If both since and userID is provided, I may make sense to accept one of them (whichever takes priority), instead of rejecting the request.`

in the jira it says: `Confirm that one of the following parameters has been provided, else returning error 400 - 001 missing/invalid parameter if not:` - hence why i throw exception